server/routes/station-csv.js

```diff
@@ -1,10 +1,19 @@
 const floodService = require('../services/flood')
 const moment = require('moment-timezone')
 const boom = require('@hapi/boom')
+const joi = require('@hapi/joi')
 
 module.exports = {
   method: 'GET',
   path: '/station-csv/{id}/{direction?}',
+  options: {
+    validate: {
+      params: joi.object({
+        id: joi.number().greater(999).less(100000).required(),
+        direction: joi.string().valid('downstream', 'upstream')
```

Comment:
I added validation to the `direction` param also. I saw a similar validation on another route, but I've also seen `direction` be just `u` or `d`.

Let me know if this is ok or adjust or remove